docs/general/events.md

```diff
@@ -18,19 +18,18 @@ The following semantic conventions for events are defined:
 ## General event attributes
 
 Events are recorded as LogRecords that are shaped
-in a special way: Event LogRecords have the attributes `event.domain`
-and `event.name` (and possibly other LogRecord attributes).
+in a special way: Event LogRecords have the attribute
+`event.name` consisting of a dot-separated `namespace` and `name` that
+uniquely identifies the event. Events may also have other LogRecord attributes.
 
-The `event.domain` attribute is used to logically separate events from different
+The `namespace` portion is used to logically separate events from different
```

Comment:
Rather than define the namespace concept in this document, I think we should link out to the [attribute naming](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-naming.md) document and say something to the effect of: "Event names are subject to the same rules as [attribute names](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-naming.md). Notably, event names are namespaced to avoid collisions."

Reply:
@jack-berg take another look pls. I also updated the general info to remove the blurbs around defining namespace concept.

Reply:
👍